### DISCUSSION TITLE
Merges to `main` show as "behind"

### DISCUSSION THREAD

**Discussion by 0x4007** [ubiquity/ubiquibot#334] [Research]
https://t.me/UbiquityDAO/29891/30084

> i’m not an expert in squash merges, but i would assume that github would be able to pick up on that and not consider it behind; it looks wrong. somebody will need to do some research to verify github’s behavior for this

> i think some indepedent research is needed, then a proposal on a fix
> for now, its up to liu to handle those merges to production

**Comment by Steveantor** [comment]
![image](https://github.com/ubiquity/ubiquibot/assets/109317559/94aee2ba-6f8e-41d2-bf34-d95f42bcbc58)
chatgpt

**Reply by 0x4007** [reply]
https://chat.openai.com/share/c8094fa1-a703-4389-9929-dd02055abae7

![photo_2023-05-27 19 33 52](https://github.com/ubiquity/ubiquibot/assets/4975670/1f8de4d7-5675-4ec4-bb72-e8428ac0d5f3)

New share feature unlocked

**Reply by Steveantor** [reply]
not on bing

**Reply by 0x4007** [reply]
<img width="1285" alt="image" src="https://github.com/ubiquity/ubiquibot/assets/4975670/415df1ba-4798-4b02-9a06-e39f4461919c">

ser bls
